packages/web-pkg/src/components/CreateLinkModal.vue

```diff
@@ -44,15 +44,14 @@
       :key="passwordInputKey"
       :model-value="password.value"
       type="password"
-      :password-policy="passwordPolicy"
+      :password-policy="!selectedLinkTypeIsInternal ? passwordPolicy : null"
```

Comment:
Don't show password policy markup and not allowed to create password on internal link at the same time